\hypertarget{PythonException_8H}{}\doxysubsection{include/jevois/\+Debug/\+Python\+Exception.H File Reference}
\label{PythonException_8H}\index{include/jevois/Debug/PythonException.H@{include/jevois/Debug/PythonException.H}}
{\ttfamily \#include $<$string$>$}\newline
{\ttfamily \#include $<$boost/python/errors.\+hpp$>$}\newline
{\ttfamily \#include $<$boost/python/object.\+hpp$>$}\newline
Include dependency graph for Python\+Exception.\+H\+:\nopagebreak
\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[width=350pt]{PythonException_8H__incl}
\end{center}
\end{figure}
This graph shows which files directly or indirectly include this file\+:\nopagebreak
\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[width=350pt]{PythonException_8H__dep__incl}
\end{center}
\end{figure}
\doxysubsubsection*{Namespaces}
\begin{DoxyCompactItemize}
\item 
 \mbox{\hyperlink{namespacejevois}{jevois}}
\begin{DoxyCompactList}\small\item\em 所有 Je\+Vois 类和函数的主命名空间 \end{DoxyCompactList}\end{DoxyCompactItemize}
\doxysubsubsection*{Functions}
\begin{DoxyCompactItemize}
\item 
std\+::string \mbox{\hyperlink{group__debugging_gaede3301d7e76cf562b3c9326b7b3d686}{jevois\+::get\+Python\+Exception\+String}} (boost\+::python\+::error\+\_\+already\+\_\+set \&)
\begin{DoxyCompactList}\small\item\em Python 异常转换为字符串，以便我们可以将回溯打印到我们的 serlog stream. \end{DoxyCompactList}\end{DoxyCompactItemize}
